client/packages/host/src/Admin/SyncSettings.tsx

```diff
@@ -38,7 +39,15 @@ const StringSyncSetting: FC<SyncSettingProps> = ({
     const patched = { ...settings, [property]: value } as SyncSettings;
     update(patched);
   };
-  return (
+  return type === 'password' ? (
+    <PasswordTextInput
+      value={value}
+      onChange={e => onChange(e.target.value)}
+      inputProps={{ autoComplete: autocomplete }}
+      disabled={disabled}
+      style={{ width: 'calc(100% - 24px)' }}
```

Comment:
to allow for the input adornment, which is adding 24px to the width in this case.